app/Components/Sudoku Board/SudokuBoard.tsx

```diff
@@ -434,11 +434,30 @@ const SudokuBoard = (props: SudokuBoardProps) => {
   };
 
   const handleKeyDown = (event: any) => {
+    const inputValue = event.nativeEvent.key;
+
+    if (inputValue == "u" || inputValue == "U") {
```

Comment:
nit: these if else if statements would work be better as a switch

Reply:
I don't mind converting it to a switch statement. I'm curious on what benefit the switch has over the if/else statement if you don't mind elaborating?

Reply:
I updated to use switch and I believe I understand the benefit of it now!